javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -91,9 +92,9 @@ protected static void addOptionalAndConstantVariables(JavaBlock function, Client
      * @param clientMethod
      * @param proxyMethodAndConstantParameters
      * @param settings
-     * @param addOptional Whether add optional variables, init to default or null
-     * @param addConstant Whether add constant variables, init to default
-     * @param ignoreParameterNeedConvert When adding optional/constant variable, ignore those which need conversion from client type to wire type. Let "ConvertClientTypesToWireTypes" handle them.
+     * @param addOptional                      Whether add optional variables, init to default or null
+     * @param addConstant                      Whether add constant variables, init to default
+     * @param ignoreParameterNeedConvert       When adding optional/constant variable, ignore those which need conversion from client type to wire type. Let "ConvertClientTypesToWireTypes" handle them.
```

Comment:
Please avoid all these unnecessary format change. It is very hard to find the actual code change.